ui/src/components/VolumeMetricGraphCard.js

```diff
@@ -139,8 +139,11 @@ const MetricGraphCard = (props) => {
   const volumeUsedOperated = operateMetricRawData(
     volumeMetricGraphData?.volumeUsed,
   );
-  const volumeLatencyOperated = operateMetricRawData(
-    volumeMetricGraphData?.volumeLatency,
+  const volumeLatencyWriteOperated = operateMetricRawData(
```

Comment:
Commit message is not very explicit about what was fixed.

You should describe what was the problem with the current code and why the new code address the problem.